src/docs/integrations/index.mdx

```diff
@@ -2,13 +2,12 @@
 title: Integrations
 ---
 
-TODO: Describe the difference between integration/plugin generations.
```

Comment:
If anyone feels strongly about leaving this in, I can, but it's bothered me for a while (this is visible, it is not a comment).